prophecies/apps/frontend/src/messages/es.json

```diff
@@ -0,0 +1,336 @@
+{
+  "actions": {
+    "cancelled": {
+      "taskRecordReview": {
+        "*": "{actor} ha cancelado sus revisiones"
+      }
+    },
+    "reviewed": {
+      "taskRecordReview": {
+        "*": "{actor} clasificó el registro con opción desconocida",
+        "choice": "{actor} clasificó el registro como <strong>\"{actionObject}\"</strong>"
+      }
+    },
+    "locked": {
+      "taskRecord": {
+        "*": "{actor} <strong>ha bloqueado</strong> el registro"
+      }
+    },
+    "unlocked": {
+      "taskRecord": {
+        "*": "{actor} <strong>ha bloqueado</strong> el registro"
+      }
+    },
+    "commented": {
+      "taskRecordReview": {
+        "*": "{actor} escribió una nota"
+      }
+    },
+    "mentioned": {
+      "user": {
+        "taskRecordReview": "{actor} mencionó <strong>{target}</strong> en una nota"
+      }
+    },
+    "selected": {
+      "taskRecordReview": {
+        "*": "{actor} seleccionó \"{alternative_value}\""
+      }
+    }
+  },
+  "adminBadge": {
+    "admin": "Panel de control"
+  },
+  "alternativeValueSelect": {
+    "selectTheCorrectValue": "Seleccione el valor correcto",
+    "useThisExactValue": "Usar el mismo valor"
+  },
+  "app": {
+    "close": "Cerrado"
+  },
+  "appHeader": {
+    "cinematicView": "Carrusel",
+    "tips": "Consejos",
+    "search": "Buscar",
+    "shortcuts": "Atajos",
+    "tutorial": "Tutorial"
+  },
+  "appSearchForm": {
+    "typeYourSearch": "Escribe tu búsqueda"
+  },
+  "appSearchResults": {
+    "tips": "Consejos"
+  },
+  "appSidebar": {
+    "dashboard": "Dashboard",
+    "myTasks": "Mis tareas",
+    "shortcuts": "Atajos",
+    "tips": "Consejos",
+    "history": "Historial",
+    "admin": "Panel de control"
+  },
+  "appVersion": {
+    "version": "Versión {version}"
+  },
+  "dashboard": {
+    "title": "Dashboard",
+    "allHistory": "Historial completo",
+    "showAllHistory": "Mostrar todo el historial",
+    "allTasks": "Todas mis tareas",
+    "allTips": "Todos los consejos",
+    "latestTips": "Últimos consejos",
+    "moreNiceTips": "Más consejos útiles",
+    "noTask": "Aún no hay tareas",
+    "lately": "Recientemente"
+  },
+  "error": {
+    "title": "Algo va mal aquí",
+    "description": "Ha ocurrido un error. Por favor, inténtelo de nuevo más tarde o póngase en contacto con su administrador si el problema persiste.",
```

Comment:
We could use the generic feminine `administradora`, it's not a standard at all but it could be cool so as not to always use the masculine term